src/main/java/org/jahia/modules/sitemap/utils/Utils.java

```diff
@@ -119,72 +119,92 @@ public static Set<String> getSitemapRoots(RenderContext renderContext, String lo
     /**
      * @return sitemap entries that are publicly accessible
      */
-    public static Set<SitemapEntry> getSitemapEntries(RenderContext renderContext, String rootPath, Locale locale) throws RepositoryException {
-        final Set<SitemapEntry> result = new LinkedHashSet<>();
+    public static void getSitemapEntries(Map<Locale, JCRSessionWrapper> sessionPerLocale, RenderContext renderContext, String rootPath, Map<Locale, Set<SitemapEntry>> entriesByLocale, Map<String, Set<SitemapEntry>> entriesByPath) throws RepositoryException {
```

Comment:
Is it possible to rename this method, a name like fillSitemapEntries is more meaningful for this method as it's not returning results anymore

Reply:
Indeed, it has been renamed. Thank you!